docs/src/content/docs/quickstart/sql-overview.mdx

```diff
@@ -0,0 +1,636 @@
+
+### Basis: re-run queries against past states without explicit snapshots
+
+Unlike in a typical SQL database, `UPDATE` and `DELETE` operations in XTDB are non-destructive, meaning previous versions of records are retained automatically and previous states of the entire database can be readily accessed.
+
+For example, despite having _seemingly_ just updated the original version of the `fred` record that was inserted initially (`SELECT * FROM people` will only show the 'current' version of everything by default), the original version was not lost and can be retrieved using a couple of methods.
+
+The simplest way to observe the prior version of the `fred` record is to re-run the exact same query against an earlier 'basis'.
+
+A basis is like a pointer to a snapshot of a previous version of the entire database state, except unlike snapshots in other systems in XTDB there is no copying or explicit snapshot creation required.
+
+A basis is stable and allows you to re-run unmodified queries indefinitely. This is useful for debugging, auditing, and exposing application data for processing in downstream systems (generating reports, analytics etc.)
+
+If you do not specify a basis then the latest available basis is used by default.
+
+Within XTDB's SQL syntax, specifying the basis can be accomplished by prefixing a query with some [special syntax](/reference/main/sql/queries) which uses the underlying notion of "DEFAULT SYSTEM_TIME" (discussed in the next section):
+
+<Tabs syncKey="psql">
+  <TabItem label="XT Play" icon="seti:db">
+    <Fiddle magicContext="my-context">
+      <Query q={`
+SETTING DEFAULT SYSTEM_TIME TO AS OF DATE '2020-01-01'
+SELECT * FROM PEOPLE`} />
+    </Fiddle>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+```sql
+user=> SETTING DEFAULT SYSTEM_TIME TO AS OF DATE '2020-01-01'
+SELECT * FROM PEOPLE;
+ _id | info | likes | name
+-----+------+-------+------
+(0 rows)
+```
+  </Card>
+  </TabItem>
+</Tabs>
+
+Here we see that with the basis now set to 2020 no results are returned.
+
+Now try setting the basis to a point in time shortly after our initial transaction:
+
+<Tabs syncKey="psql">
+  <TabItem label="XT Play" icon="seti:db">
+    <Fiddle magicContext="my-context">
+      <Query q={`
+SETTING DEFAULT SYSTEM_TIME TO AS OF DATE '2024-01-01' -- try adding 3 days
+SELECT * FROM PEOPLE`} />
+    </Fiddle>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+  <Aside type="caution">The timestamp shown here is for illustration only, you should refer to the output of your recent transactions query for a suitable timestamp</Aside>
```

Comment:
I think we need a little bit of "why" here:
```
When querying against a real XTDB node (like you are), XTDB sets the `SYSTEM_TIME` of `INSERT`/`UPDATE`s to the current time.

So in the below query substitute the date for dates relative to what you see in the `_txs` table.
```

Maybe we could even have the example date be `TIMESTAMP '<your date>'`

Reply:
> Maybe we could even have the example date be TIMESTAMP '<your date>'

Opted against this on the basis that the psql examples already otherwise do contain lots of ~arbitrary example timestamps. It also makes it motivating to get Play running real psql at some point :slightly_smiling_face: